<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSurveyMethodWnd</class>
 <widget class="QWidget" name="MSurveyMethodWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MSurveyMethodWnd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_9">
     <property name="text">
      <string>磁力仪</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_8">
     <property name="text">
      <string>GPS/全站仪</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_7">
     <property name="text">
      <string>采泥器</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_6">
     <property name="text">
      <string>声速仪</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_5">
     <property name="text">
      <string>验潮仪</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="text">
      <string>侧扫声呐</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>ADCP验流</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>单波束数据</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>多波束数据</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
